GVFS/GVFS/CommandLine/SparseVerb.cs

```diff
@@ -599,45 +596,52 @@ private void CheckGitStatus(ITracer tracer, GVFSEnlistment enlistment, HashSet<s
                 }
                 else
                 {
-                    this.WriteMessage(tracer, statusResult.Output);
-                    this.WriteMessage(tracer, "git status reported that you have dirty files");
-                    this.WriteMessage(tracer, "Either commit your changes or reset and clean");
+                    this.WriteMessage(tracer, "git status reported that you have dirty files:\n");
+                    foreach (string path in dirtyUncoveredPaths)
+                    {
+                        this.WriteMessage(tracer, $"    {path}");
+                    }
+
+                    this.WriteMessage(tracer, "\nEither commit your changes or reset and clean");

```

Comment:
Should these `\n` newlines be special-cased depending on platform? Or should we write an empty message to create the blank lines?

Reply:
> Should these \n newlines be special-cased depending on platform? 

Good point, iI can update these to `Environment.NewLine`.

> Or should we write an empty message to create the blank lines?

We could also write a blank line to the console, but I don't think we'd want to call `this.WriteMessage` as it writes to the log file as well (and it might look a bit weird to have the blank messages there).  We do have the option of just writing a new line to the console directly, but `Environment.NewLine` avoids mixing `WriteMessage` with calls to write directly to the console.

Reply:
Next push will be a force-push to replace to fix up this first commit, I'll then add a second commit with functional test updates.

Reply:
Maybe construct the full multi-line string in a stringbuffer then write them at once in a single WriteMessage call?

Reply:
That's a good idea, thanks!